src/js/components/products/detail.tsx

```diff
@@ -84,15 +140,53 @@ const ProductDetail = ({ product, productSlug, doFetchObject }: Props) => {
             className="slds-col
               slds-size_1-of-1
               slds-medium-size_2-of-3
-              slds-p-bottom_x-large
-              slds-text-longform"
+              slds-p-bottom_x-large"
           >
-            <Button
-              label={i18n.t('Create a Project')}
-              className="slds-size_full slds-p-vertical_xx-small"
-              variant="brand"
-              disabled
-            />
+            {!projects || !projects.fetched ? (
+              // Fetching projects from API
+              <Spinner />
+            ) : (
+              <>
+                <ProjectForm
+                  product={product}
+                  startOpen={!projects.projects.length}
+                />
+                {Boolean(projects.projects.length) && (
```

Comment:
You can probably just let this get implicitly coerced to boolean.

Reply:
We could, if the code were adjusted to `{ projects.projects.length ? (...) : null}`. That is, using a ternary operator instead of a logical and operator. I don't have strong preferences on which is cleaner/clearer.